diff --git a/app/page.tsx b/app/page.tsx
index 0939296..9de4b38 100644
--- a/app/page.tsx
+++ b/app/page.tsx
@@ -10,7 +10,11 @@ export default function Home() {
                 <div className="p-6 text-center">
                     <div className="relative mx-auto mb-4 size-[150px] overflow-hidden rounded-full bg-gray-100">
                         <Image
-                            src="/images/lm-profile.jpg"
+                            src={
+                                process.env.NODE_ENV == "development"
+                                    ? "/images/lm-profile.jpg"
+                                    : "/lm/images/lm-profile.jpg"
+                            }
                             alt="Lisa-Marie Schwingshandl"
                             fill
                             sizes="150px"
diff --git a/next.config.js b/next.config.js
index 7d14b36..1d934b4 100644
--- a/next.config.js
+++ b/next.config.js
@@ -1,5 +1,6 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
+    basePath: process.env.NODE_ENV == "development" ? "" : "/lm",
     experimental: {
         reactCompiler: true,
     },
